web_poet/fields.py

```diff
@@ -118,33 +96,47 @@ def get_fields_dict(cls_or_instance) -> Dict[str, FieldInfo]:
     return getattr(cls_or_instance, _FIELDS_INFO_ATTRIBUTE_READ, {})
 
 
-async def item_from_fields(obj, item_cls=dict, *, item_cls_fields=False):
+T = TypeVar("T")
```

Comment:
Should we utilize the same `ItemT` from `web_poet/pages.py`? as they're essentially the same

Reply:
That's an interesting point, I have't considered it. Are there benefits in reusing typevars like this? 

In this particular case it seems there are scenarios when T can be not the same as ItemT, so they shouldn't be reused.

Reply:
No real benefit aside from ensuring that the types are consistent. But if some scenarios would result in diverging types, then we shouldn't reuse them. 👍